.github/workflows/release.yml

```
@@ -43,7 +45,8 @@ jobs:
       # -
       #   name: Publish
       #   uses: actions/publish-immutable-action@v0.0.4
-
+  
+  type=raw,${{ env.IMAGE_UBUNTU }}
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 49: could not find expected ':'

(syntax-check)